test/Test/DecisionSpec.hs

```diff
@@ -1,6 +1,8 @@
-module Test.DecisionSpec where
+module Test.DecisionSpec
+    (genDecision)
+    where
```

Comment:
This export is not formatted properly. And moreover, now tests from this module are not running since you don't export them.